[32mINFO    [0m root:__init__.py:27 功能的名称为：test_ryvue 功能的描述为：None
[31m[1mERROR   [0m root:test_ryvue.py:70 login_010断言失败，用例数据为：{'username': 'ADMIN', 'password': 'admin123'}期望数据为：{'msg': '用户不存在/密码错误', 'code': 500}服务器返回的期望数据：{"msg":"操作成功","code":200,"token":"eyJhbGciOiJIUzUxMiJ9.eyJsb2dpbl91c2VyX2tleSI6Ijg5NzUyZjY0LTRmNGItNDVkZS1iZjc5LTcxMTE5NWFlN2RhMyJ9.7qLdX5QmWeG7DhrRBiZrSEwXN4ODW6VoFEdLH1X-rULrONOBW_XgigEBp8NjtLDgb3kiJJICXLl2YWfmPcBiRA"}
[31m[1mERROR   [0m root:__init__.py:33 断言失败